python/lsst/summit/utils/utils.py

```diff
@@ -26,6 +26,9 @@
 import lsst.pipe.base as pipeBase
 import lsst.utils.packages as packageUtils
 from lsst.daf.butler.cli.cliLog import CliLog
+import datetime
+from datetime import timedelta
```

Comment:
Do you need to import this separately if you already have `datetime` imported?

Reply:
Thanks, I've fixed that, and also added the functions to `__all__` which I'd forgotten to do, and reformatted the `__all__` block to make future changes easier to diff.